docs/pages/database-access/enroll-aws-databases/sql-server-ad.mdx

```diff
@@ -18,6 +18,11 @@ videoBanner: k2wz79XCexY
 This guide will focus on Amazon RDS for SQL Server using AWS-managed Active
 Directory authentication.
 
+This guide shows how to enroll a single Microsoft SQL Server database through
+the Teleport Database Service. For a more scalable approach, learn how to
+[configure the Teleport Discovery Service](../../auto-discovery/databases.mdx)
+to automatically enroll all AWS databases in your infrastructure.
```

Comment:
```suggestion

```

There is no auto-discovery for RDS SQL server with AD.

(not to be confused with RDS Proxy for SQL Server).